github/repos.go

```diff
@@ -538,6 +540,12 @@ type RequiredPullRequestReviews struct {
 	IncludeAdmins bool `json:"include_admins"`
```

Comment:
The `RequiredStatusChecks.IncludeAdmins` field is now documented both as required and deprecated.

It's pretty puzzling to me how to use the new API. There are two `bool`s, do they need to be set to the same value if `true`, or does `EnforceAdmins` take higher precedence over `RequiredStatusChecks.IncludeAdmins`? Or can either one be set to `true`?